specs/ldm/link-discovery-management-spec.html

```diff
@@ -303,32 +316,39 @@ <h3>Terminology</h3>
         <dl class="spaced">
           <dt><dfn>Link</dfn></dt>
           <dd>
-            An assertion or RDF triple consisting of a (subject, predicate, object) that manifests an instance of a relationship between the referenced subject and object.
+            An assertion or RDF triple consisting of a (subject, predicate, object) that manifests an instance of a
```

Comment:
OSLC has Link class, which is also a valid representation of a link as a whole resource.

Reply:
not aware of that and couldn't find any reference. Could you please provide the source for "oslc:link"?